edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -0,0 +1,95 @@
+var edx = edx || {};
+
+(function($) {
+  'use strict';
+
+  var actionToMessageTypesMap = {
+    'submit': {
+      promptEventName: 'endExamAttempt',
+      responseEventName: 'examAttemptEnded'
+    },
+    'start': {
+      promptEventName: 'startExamAttempt',
+      responseEventName: 'examAttemptStarted'
+    }
+  };
+
+  function workerPromiseForEventNames(eventNames) {
+    return function() {
+      var proctoringBackendWorker = new Worker(edx.courseware.proctored_exam.configuredWorkerURL);
```

Comment:
may want to check for window.Worker... although if they've gotten this far on a browser that doesn't support web workers I'd be impressed :)

Reply:
Nevermind, I see further down you check in `shouldUseWorker`